actions/createList.js

```diff
@@ -1,96 +1,65 @@
-import fs from 'fs';
-import path, { dirname } from 'path';
-import { fileURLToPath } from 'url';
+import fs from "fs";
+import path, { dirname } from "path";
 
-function getAllFiles(dirPath, arrayOfFiles) {
-  let files = fs.readdirSync(dirPath)
+const ROOT_PATH = path.resolve("./chains");
+const CHAIN_INFO_FILE = "chain-information.json";
+const CHAIN_LOGO_FILE = "chain-logo.png";
+const NATIVE_TOKEN_LOGO_FILE = "token-logo.png";
+const CONTRACT_TOKEN_INFO_FILE = "contract-information.json";
+const CHAINS_FOLDER_URL =
+  "https://github.com/ava-labs/subnet-assets/tree/main/chains";
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const TOKEN_LIST_FILE = path.resolve(__dirname, "../tokenList.json");
+import { fileURLToPath } from "url";
 
-  arrayOfFiles = arrayOfFiles || []
-
-  files.forEach(function (file) {
-    if (fs.statSync(dirPath + "/" + file).isDirectory()) {
-      arrayOfFiles = getAllFiles(dirPath + "/" + file, arrayOfFiles)
-    } else if (path.extname(file) == '.json') {
-      arrayOfFiles.push(path.join(dirPath, "/", file))
-    }
-  })
-
-  return arrayOfFiles
+function getTokens(chainId, chainTokenIds) {
+  return fs
```

Comment:
Since wallet just needs ERC20 tokens, we're going to need to filter for contracts where `contractType` is `ERC20` here.

Reply:
If we filter by ERC20, it probably makes sense to move the `tokenList.json` into a directory `tokenLists` and name it `erc20TokenList.json` or something.  I'm certain that a need for more token lists will arise and this will make it easier to scale.